e2e/cypress/integration/channel/header_spec.js

```diff
@@ -102,3 +82,19 @@ describe('Header', () => {
         cy.get('#searchBox').should('be.visible').and('be.empty');
     });
 });
+
+function updateAndVerifyChannelHeader(prefix, header) {
+    // # Update channel header
+    cy.updateChannelHeader(prefix + header);
+
+    // * Should render blockquote if it starts with ">"
+    if (prefix === '>') {
+        cy.get('#channelHeaderDescription > span > blockquote');
```

Comment:
This step does not do anything. I guess you are missing a should('be.visible') condition here.

Reply:
Oh, you're right.  Will update that.

Reply:
@srkgupta Now updated, thanks